python/graphstorm/trainer/glem_np_trainer.py

```diff
@@ -116,7 +117,7 @@ def fit(self, train_loader, num_epochs,
                 self._model.lm.unfreeze_input_encoder()
                 no_pl = False
 
-            use_gnn = not self._model.em_order_gnn_first
+            use_gnn = self._model.em_order_gnn_first
```

Comment:
why do you change the order?

Reply:
The previous implementation was wrong: conflicts with what the flag suggests.